tests/p2p/test_split_brain.py

```diff
@@ -46,6 +47,8 @@ def create_peer(self, network, unlock_wallet=True):
         wallet.unlock(words=words, tx_storage=manager.tx_storage)
         return manager
 
+    # retry the test once if it fails, see: https://github.com/box/flaky
+    @pytest.mark.flaky
```

Comment:
How do you configure how many attempts/failures you want? Is it always one out of two?

Reply:
The Flaky GitHub says it's `@flaky(max_runs=3, min_passes=2)`, I suppose it also works as `@pytest.mark.flaky(max_runs=3, min_passes=2)`, but I haven't tested it.